mlir/test/fusion/e2e/mixr-gemm/mixr-gemm-bcast-add.e2e.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: rocmlir-opt -migraphx-to-tosa %s | rocmlir-driver -host-pipeline highlevel | rocmlir-gen -ph -print-results -rand none - | rocmlir-driver -c  | mlir-rocm-runner --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s
```

Comment:
One thing I've been considering doing with adding gemm testing is adding the xmir runner pipeline as an option to `rocmlir-driver` so that we can use xmir stuff in tests without having to pay the compilation overhead inside the "runner"

(and I've also been contemplating making `-c` be "full" for kernel pipelines and stripping out most of mlir-rocm-runner's stuff)

Reply:
(and I don't think we've had --verifier clone in the tests when this PR landed?)